kr/kr_pgp.go

```diff
@@ -322,3 +324,98 @@ func uninstallCodesigning() {
 	exec.Command("git", "config", "--global", "--unset", "commit.gpgSign").Run()
 	exec.Command("git", "config", "--global", "--unset", "tag.forceSignAnnotated").Run()
 }
+
+func pgpSignCommand(c *cli.Context) (err error) {
+	message := c.String("message")
+	sigType := c.String("sigType")
+	if sigType != "detach" && sigType != "attach" && sigType != "clearsign" {
+		err = errors.New("Invalid signature type!\n")
+		os.Stderr.WriteString(err.Error())
+		return
+	}
+
+	request, err := kr.NewRequest()
+	if err != nil {
+		os.Stderr.WriteString(err.Error())
+		return
+	}
+	startLogger(request.NotifyPrefix())
+	request.BlobSignRequest = &kr.BlobSignRequest{
+		Blob:    message,
+		SigType: sigType,
+	}
+
+	response, err := requestBlobSignature(request)
+	if err != nil {
+		return
+	}
+	sig, err := response.AsciiArmorSignature()
+	if err != nil {
+		os.Stderr.WriteString(err.Error())
+		return
+	}
+	os.Stdout.Write([]byte("\n"))
```

Comment:
This newline can be printed to `Stderr` since it is just for formatting. Otherwise piping to another command prefixes the signature with a newline.